app/src/main/java/com/example/weatherapp/ui/main/MainScreenViewModel.kt

```diff
@@ -33,7 +33,7 @@ class MainScreenViewModel @Inject constructor(
     val forecastData = MutableLiveData<ForecastWeather>()
     val progress = MutableLiveData<Boolean>(false)
     val cityName = MutableLiveData<String>()
-
+    var temp: LiveData<String> = Transformations.map(weatherData) { weather -> weather.main.temp.toString() + " C" }
```

Comment:
it can be val. and use Kotlin String concatation like "Hello, ${viewModel.world}"

Reply:
temperature is more readable